content/pages/disability-benefits/claims-appeal/request-a-priority-review.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page-breadcrumbs.html
+title: Request a Priority Review
+display_title: Request a Priority Review
+order: 1
+plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
+template: detail-page
+---
+<div itemscope itemtype="http://schema.org/HowTo">
+
+<div class="va-introtext" itemprop="description">
+If you need to have your appeal moved to the front of the line due to serious illness, financial distress, or other reasons, find out how to request a priority review.
+</div>
+
+<h3>Is there a way to prioritize my appeal?</h3>
+Advanced on Docket appeals are prioritized so that they are always at the front of the line. You can apply to have your appeal Advanced on Docket for any of the below reasons.
+
+<div class="feature">
+
+**One of these must be true. You:**
+- Are 75 years or older, **or**
+- Suffer from a serious illness, **or**
+- Are in financial distress, **or**
+- Have other sufficient cause (a reason for needing your appeal reviewed faster)
+
+**Note:** If you are older than 75, your appeal will receive Advanced on Docket status automatically.
```

Comment:
:thinking: So is it >= 75, or just == 75?

Previously, we say "75 or older", and here it's just "older than 75."

Reply:
Could you change this to "75 or older"